### PR TITLE
Adapt to 0.7 Kyma release

### PR DESCRIPTION
**Pull request by sjanota**
In Kyma 0.7 newer kubernetes and Minikube are used. This PR bumps this in CLI code and adapts to new versions of those tools.

As a bonus following bugs are fixed:
#20 
#19 
Problem with displaying installation steps

### CODE REVIEW

File: pkg/kyma/cmd/provision/minikube.go [hunk 1]
```diff
@@ -129,30 +130,34 @@ func (o *MinikubeOptions) Run() error {
...
 		s.Failure()
 		return err
 	}
 
-	err = internal.WaitForPod("kube-system", "k8s-app", "kube-dns")
-	if err != nil {
-		s.Failure()
-		return err
-	}
+	//s.Status("Await kube-dns to be up and running")
```
Comment (a-thaler) [inline pkg/kyma/cmd/provision/minikube.go]: Can you please decide if it is needed or not and then keep or delete the code.

File: pkg/kyma/cmd/provision/minikube.go [hunk 2]
```diff
@@ -179,16 +184,12 @@ func (o *MinikubeOptions) Run() error {
 }
 
 func checkIfMinikubeIsInitialized(o *MinikubeOptions) error {
-	statusText, err := internal.RunMinikubeCmdE([]string{"status", "-b=" + bootstrapper, "--format", "'{{.MinikubeStatus}}'"})
```
Comment (a-thaler) [inline pkg/kyma/cmd/provision/minikube.go]: Shouldn't we keep the bootstrapper argument? yes, we are using the default, but then we should delete it from start command as well and delete the variable..

File: pkg/kyma/cmd/provision/minikube.go [hunk 3]
```diff
@@ -233,27 +233,23 @@ func initializeMinikubeConfig() error {
...
 	startCmd := []string{"start",
 		"--memory", o.Memory,
 		"--cpus", o.CPU,
-		"--extra-config=apiserver.Authorization.Mode=RBAC",
-		"--extra-config=apiserver.GenericServerRunOptions.CorsAllowedOriginList='.*'",
-		"--extra-config=controller-manager.ClusterSigningCertFile='/var/lib/localkube/certs/ca.crt'",
-		"--extra-config=controller-manager.ClusterSigningKeyFile='/var/lib/localkube/certs/ca.key'",
-		"--extra-config=apiserver.admission-control='LimitRanger,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota'",
+		"--extra-config=apiserver.authorization-mode=RBAC",
+		"--extra-config=apiserver.cors-allowed-origins='http://*'",
```
Comment (a-thaler) [inline pkg/kyma/cmd/provision/minikube.go]: Will it work for https?

File: pkg/kyma/cmd/provision/minikube.go (hunk 1 above)
Comment (sjanota) [reply pkg/kyma/cmd/provision/minikube.go]: What I'm sure is it has changed in new minikube. What I can do is write it the same way installation scripts are written (they work on labels which haven't changed).

File: pkg/kyma/cmd/provision/minikube.go (hunk 2 above)
Comment (sjanota) [reply pkg/kyma/cmd/provision/minikube.go]: I'll add it back

File: pkg/kyma/cmd/provision/minikube.go (hunk 3 above)
Comment (sjanota) [reply pkg/kyma/cmd/provision/minikube.go]: Shouldn't be a problem, apiserver is not hidden by SSL anyway. That's how Kyma is being installed right now.

File: pkg/kyma/cmd/provision/minikube.go (hunk 1 above)
Comment (sjanota) [reply pkg/kyma/cmd/provision/minikube.go]: I thought the problem was with renaming in minikube, but the problem was with IsPodReady function. Fix is en route.

Comment (a-thaler) [reply pkg/kyma/cmd/provision/minikube.go]: jihaa, thanks

**Pull request merged**